### DISCUSSION TITLE
CHAT

### DISCUSSION THREAD

**Discussion by Z-YHWH-24** [Z-YHWH-24/ENEO-AI-PROJET#1] [General]
HELLO  EVERY ONE

**Comment by Z-YHWH-24** [comment]
Qui est la ?

**Reply by audreyossalobo** [reply]
moi

**Reply by Z-YHWH-24** [reply]
et les autres tu peut faire pression de  ton coter stp

**Reply by audreyossalobo** [reply]
mais pas pour longtemps  svp je venais a peine de rentrer je ne me suis meme pas encore reposer

**Reply by Z-YHWH-24** [reply]
a 23h ?

**Comment by audreyossalobo** [comment]
Ah je fais comment Cc

Le mar. 30 juil. 2024 à 00:24, Z-YHWH-24 ***@***.***> a
écrit :

**Reply by Z-YHWH-24** [reply]
je voulais qu'on assigne les pages a chacun mais  bon.......vue que t'es la quel interface te plait que tu vas te sentir a l'aise de developper ?

**Reply by Z-YHWH-24** [reply]
entre temps j'ai mal cree les branche je vais refaire sa tout de suite la.......en attendant choisis deja l'interface que tu veut je crois que tout le  monde a les templates n'est ce pas ?

**Comment by audreyossalobo** [comment]
mom 21h

**Comment by audreyossalobo** [comment]
Oui oui

Le mar. 30 juil. 2024 à 00:29, Z-YHWH-24 ***@***.***> a
écrit :

**Reply by Z-YHWH-24** [reply]
une fois choisis tu ecris signale ici je vais cree une autre discusiion pour signaler la bas uniquement pour signaler svp

**Reply by Ornella-Ashley** [reply]
ok

**Comment by audreyossalobo** [comment]
d'accord

**Comment by audreyossalobo** [comment]
on insert les photos comment sur le choix

**Reply by Z-YHWH-24** [reply]
en bas de chaque commentaire que tu veut faire tu vas voir "paste ,  drop  ,   or click to add files " tu clique tu importe

**Comment by Z-YHWH-24** [comment]
qui est la ?

**Comment by Z-YHWH-24** [comment]
je vais supprimer certains fichier puis je vais crees les dossier dans lesquels tu vous aller mettre vos fichiers
